src/shared/sessionStorage.ts

```diff
@@ -4,7 +4,7 @@ import { MinimalStorage } from 'utils/storage/MinimalStorage'
 
 export const SESSION_KEY = '_hvsession'
 export const KA_SESSION_KEY = '_hvsessionexpiry'
-const SESSION_EXPIRY_TIMEOUT = 30 * 60 * 1000
+const SESSION_EXPIRY_TIMEOUT = 3 * 60 * 60 * 1000
```

Comment:
⏳ 👌

Reply:
How did we decide on 3 hours? (Just curious, feels like a good session length to me 👍 )

Reply:
Haha, there's a very healthy process behind it 😅

1. "Wat, we only have 30 minutes? 🤨"
2. "We should probably change that 🤔"
3. "Lets 10x it and see what happens 🤷‍♂️"
3. "Lol that's not 10x, but 3 hours seems good so ima keep that 👊"